department-of-veterans-affairs/vets-api
# STEM AD: Queue solution #17771

## Description of change

Add sidekiq job that runs once an hour between 6-18, this is based on EVSS availability.
This job will take all 10203 submissions with a logged in user and apply business logic to them to determine if in the spool file should have header of V10203 or 10203DNY for processing in TIMS

## Original issue(s)
department-of-veterans-affairs/va.gov-team#17771

## Things to know about this PR

Test cases
1. Submit another form than 10203
    - expected that a spool file is generated for this form
2. Submit a 10203s based on AC in story

![Screen Shot 2021-01-12 at 12 20 12 PM](https://user-images.githubusercontent.com/1094999/104349131-88cd0380-54d0-11eb-9f48-923e9046df23.png)
